test/v3/controllers/SubmitSelfEmploymentBsasControllerSpec.scala

```diff
@@ -181,10 +179,11 @@ class SubmitSelfEmploymentBsasControllerSpec
         val auditResponse: AuditResponse =
           AuditResponse(
             httpStatus = BAD_REQUEST,
-            errors = Some(Seq(
-              AuditError(NinoFormatError.code),
-              AuditError(CalculationIdFormatError.code)
-            )),
+            errors = Some(
+              Seq(
+                AuditError(NinoFormatError.code),
+                AuditError(CalculationIdFormatError.code)
+              )),
```

Comment:
below the `multiple errors occur for the customised errors` needs updating. `RuleAdjustmentRangeInvalid` and `FormatAdjustmentValueError` can be taken out